tutorials/misc/plot_ecog.py

```diff
@@ -9,71 +11,152 @@
+# This is best accomplished with matplotlib. We can capture an image of the
+# current mayavi view, along with the xy position of each electrode, with the
+# `snapshot_brain_montage` function. We can then visualize for example the
+# gamma power on the brain using MNE and matplotlib functions.
+# Here, we'll use :func:`~mne.viz.snapshot_brain_montage` to save the plot
+# as image data, so that later in the tutorial we can plot data on top of it.
 
-subjects_dir = mne.datasets.sample.data_path() + '/subjects'
-fig = plot_alignment(info, subject='sample', subjects_dir=subjects_dir,
+subjects_dir = sample_path + '/subjects'
+fig = plot_alignment(raw.info, subject='sample', subjects_dir=subjects_dir,
                      surfaces=['pial'])
 mne.viz.set_3d_view(fig, 200, 70)
 
+xy, im = snapshot_brain_montage(fig, montage)
+
 ###############################################################################
-# Sometimes it is useful to make a scatterplot for the current figure view.
-# This is best accomplished with matplotlib. We can capture an image of the
-# current mayavi view, along with the xy position of each electrode, with the
-# `snapshot_brain_montage` function.
+# We then compute the signal power in certain frequency bands (e.g. 30-90 Hz).
+# We compute the power in gamma and alpha bands.
+gamma_power_t = raw.copy().filter(30, 90).apply_hilbert(
+    envelope=True).get_data()
+alpha_power_t = raw.copy().filter(8, 12).apply_hilbert(
+    envelope=True).get_data()
+gamma_power = gamma_power_t.mean(axis=-1)
+alpha_power = alpha_power_t.mean(axis=-1)
 
-# We'll once again plot the surface, then take a snapshot.
-fig_scatter = plot_alignment(info, subject='sample', subjects_dir=subjects_dir,
-                             surfaces='pial')
-mne.viz.set_3d_view(fig_scatter, 200, 70)
-xy, im = snapshot_brain_montage(fig_scatter, montage)
+###############################################################################
+# Now let's use matplotlib to overplot frequency band power onto the electrodes
+# which can be plotted on top of the brain from `snapshot_brain_montage`.
 
 # Convert from a dictionary to array to plot
-xy_pts = np.vstack([xy[ch] for ch in info['ch_names']])
+xy_pts = np.vstack([xy[ch] for ch in raw.info['ch_names']])
+
+# colormap to view spectral power
+cmap = 'viridis'
+
+# create a 1x2 subplot showing a snapshot of the average power
+# in gamma and alpha frequency bands
+fig, axs = plt.subplots(1, 2, figsize=(20, 10))
+_gamma_alpha_power = np.concatenate((gamma_power, alpha_power)).flatten()
+vmin, vmax = np.percentile(_gamma_alpha_power, [10, 90])
+for ax, band_power, band in zip(axs,
+                                [gamma_power, alpha_power],
+                                ['Gamma', 'Alpha']):
+    ax.imshow(im)
+    ax.set_axis_off()
+    sc = ax.scatter(*xy_pts.T, c=band_power, s=200,
+                    cmap=cmap, vmin=vmin, vmax=vmax)
+    ax.set_title(f'{band} frequency', size='large')
+fig.colorbar(sc, ax=ax)
```

Comment:
```suggestion
    ax.set_title(f'{band} band power', size='x-large')
fig.colorbar(sc, ax=axs)
```
Using `axs` here will steal space *equally from both subplots* to make room for the colorbar.